buildSrc/src/main/kotlin/Prometheus.kt

```diff
@@ -0,0 +1,4 @@
+object Prometheus {
+    const val SimpleClientCommon = "io.prometheus:simpleclient_common:0.16.0"
```

Comment:
For å pirke litt, så kan du godt leggje inn avhengnad til denne fila frå app-etterlatte-<appnamn>.yaml-filene som bruker denne. Da blir det automatisk trigga nye bygg av dei når vi endrar vesjonar her.

Reply:
Men - avhengigheten trengs i etterlatte-ktor lib - ikke apps ? Eller fungere det for lib/etterlatte-app- på samme måte som apps/etterlatte-app-?

Reply:
Ser at de andre med etterlatte- som navn er i libs - så regner med det fungere. Kan omdøpe filen

Reply:
Kanskje det held å få den til å vera avhengig av etterlatte-ktor, og så sørge for at etterlatte-ktor blir bygd på nytt ved endring her? Hadde i så fall vore enda betre.

Reply:
Fikk en forklaring fra Mads - jobben er nå gjort.